lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
```

Comment:
@revgum I'm on the fence as to how best to set up this role. We could do it super early, like when a user is created, and/or as a background job when mediated deposit is enabled. We could also do it on demand, like: when the reviewer sends a submission back to the depositor, we can call an action that ensures the depositor has the right role to resubmit their deposit?

Thoughts, @jcoyne @jeremyf?